src/app/pages/apps/components/app-delete-dialog/app-delete-dialog.component.html

```diff
@@ -0,0 +1,41 @@
+<h1 matDialogTitle>
+  {{ 'Delete' | translate }}
+</h1>
+<form class="ix-form-container" [formGroup]="form" (submit)="onSubmit()">
+  <p class="message">
+    {{ 'Delete {name}?' | translate: { name: data.name } }}
+  </p>
+
+  @if(data.showRemoveVolumes) {
+    <ix-checkbox
+      formControlName="remove_volumes"
+      [label]="'Remove iXVolumes' | translate"
+    ></ix-checkbox>
+  }
+
+  <ix-checkbox
+    formControlName="remove_images"
+    [label]="'Remove Images' | translate"
+  ></ix-checkbox>
+
+  <ix-checkbox
```

Comment:
I think we can remove the confirmation checkbox since it's already a part of confirmation dialog.